src/cascadia/TerminalCore/TerminalApi.cpp

```diff
@@ -36,6 +36,11 @@ void Terminal::SetTextAttributes(const TextAttribute& attrs) noexcept
     _buffer->SetCurrentAttributes(attrs);
 }
 
+Viewport Terminal::GetBufferSize() noexcept

```

Comment:
I'd almost rather that this was just `GetBufferWidth`, since that's all we're using it for. Getting the whole buffer size might be something that the terminal api need not expose

Reply:
I see what you mean.

I'm mostly copying code from `SCREEN_INFORMATION`:

https://github.com/microsoft/terminal/blob/74909c0c6511787907a6a369c8046c11ee373ae8/src/host/screenInfo.cpp#L145

Does this somewhat justify `GetBufferSize` here?,,

Reply:
The buffer has the viewport size sitting there in a member variable anyway so I don't see the need to restrict this to just width for perf reasons. I think it's fine enough.